server/fleet/app.go

```diff
@@ -210,31 +211,6 @@ type VulnerabilitiesWebhookSettings struct {
 	HostBatchSize int `json:"host_batch_size"`
 }
 
-// JiraIntegration configures an instance of an integration with the Jira
-// system.
-type JiraIntegration struct {
```

Comment:
This was just moved to a new `server/fleet/integrations.go` file to have all integrations-related structs and funcs in one place.